x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml

```diff
@@ -23,6 +23,9 @@ setup:
               metricset:
                 type: keyword
                 time_series_dimension: true
+              count:
```

Comment:
I don't see this field being used anywhere in this test. Is this a left over?

Reply:
I added it because I saw there was no metric of type `counter`. Will add a few values and test it together with `gauge`.